addons/miscFixes/patchRHSAFRF/config.cpp

```diff
@@ -50,4 +51,10 @@ class CfgWeapons {
             };
         };
     };
+    class rhs_medic_bag: B_AssaultPack_Base {};
+    class rhs_medic_bag_XL: rhs_medic_bag {
```

Comment:
backpacks will be a cfgVeh, unless this is a vest? will check

Reply:
You are, in fact, correct! Pushing a fix now.